From 952de80e561668fd4503cf52e56c97d24ebcb341 Mon Sep 17 00:00:00 2001
From: CI Assistant <yossi.ilkanaev@intel.com>
Date: Thu, 24 Jan 2019 16:43:14 +0100
Subject: [PATCH] Merge pull request #430 in SW_UGW/linux from
 bugfix/PMCPESYS-126-ugw8-revert-eth_cooling-support-in-thermal-driver-ltq_thermal.c
 to master

* commit 'b903294a363e3fb609ff480f483466d6562499d2':
  revert changes to support eth_cooling device
---
 drivers/thermal/ltq_thermal.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/thermal/ltq_thermal.c b/drivers/thermal/ltq_thermal.c
index 4a9b22f3d612..c1fc4e1b5576 100644
--- a/drivers/thermal/ltq_thermal.c
+++ b/drivers/thermal/ltq_thermal.c
@@ -272,10 +272,14 @@ static int ltq_thermal_get_trend(void *data, int trip,
 
 	if (temp > last_temp)
 		*trend = THERMAL_TREND_RAISING;
+	else if (temp < last_temp && temp <= (trip_temp - trip_hyst))
+		*trend = THERMAL_TREND_DROPPING;
+/*
 	else if (temp < last_temp && temp > (trip_temp - trip_hyst))
 		*trend = THERMAL_TREND_DROPPING;
 	else if (temp < last_temp && temp <= (trip_temp - trip_hyst))
 		*trend = THERMAL_TREND_DROP_FULL;
+*/		
 	else
 		*trend = THERMAL_TREND_STABLE;
 
